### ISSUE TITLE
Cannot record previously recorded shortcut

### ISSUE THREAD

**Issue by p0deje** [sindresorhus/KeyboardShortcuts#15]
I've found a small issue when integrating the package. It can also be reproduced in Lungo app:

1. Open "Preferences" -> "Keyboard Shortcuts..."
2. Register CMD+SHIFT+K shortcut.
3. Register CMD+SHIFT+L shortcut.
4. Try to register CMD+SHIFT+K shortcut again.

For some reason, it's no longer possible to register CMD+SHIFT+K shortcut until the application is restarted.

**Comment by sindresorhus** [comment]
This should have been fixed by https://github.com/sindresorhus/KeyboardShortcuts/commit/56778e11a762931feebbdb726269ddf3919bc084. I just confirmed that the problem existed before that commit and I cannot reproduce it after. Lungo is still on an older version of KeyboardShortcuts.

**Comment by sindresorhus** [comment]
I noticed https://github.com/p0deje/Maccy/commit/1001035d611cb411df702a941f5eb40626fc5ea7. That's awesome! Feel free to open issue for anything else that might have simplified switching over.

I think https://github.com/sindresorhus/KeyboardShortcuts/issues/1 would be helpful for you. If you have any thoughts on that, feel free to comment. I think you also might find https://gist.github.com/sindresorhus/2bb90276ad608a22ee5e8fb291b35b88 useful. I saw you're using those Carbon APIs, but that gist wraps them up into a nicer API.
